docs/generate_checks.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Security Scorecard Authors
+#
```

Comment:
Same question as another comment on the PR - any reason to use Python instead of Go? No need to update the language, just curious if there was any reasoning behind the choice.